woocommerce/woocommerce
# Remove wp-env workaround

### Changes proposed in this Pull Request:

This PR removes the workaround introduced in https://github.com/woocommerce/woocommerce/pull/37587 since the bug has been fixed latest version of wp-env.

### How to test the changes in this Pull Request:

1. Checkout this branch 
2. Run E2E tests and confirm they all pass using `pnpm --filter=woocommerce run env:test && pnpm  --filter=woocommerce run test:e2e-pw`
3. Run API tests and confirm they all pass using `pnpm --filter=woocommerce run env:test && pnpm  --filter=woocommerce run test:api`.

I didn't add instructions for testing performance tests locally, so I'm relying on the actions attached to this PR to demonstrate that they are still working as expected.
